ddtrace/contrib/integration_registry/registry.yaml

```diff
@@ -393,6 +393,16 @@ integrations:
       min: 20.12.1
       max: 24.11.1
 
+- integration_name: gunicorn
```

Comment:
is this a relevant change?

Reply:
This was a bug we missed because of how the integration_registry test was written. The test looked for integrations in ddtrace/contrib/internal/{integration_name}, but Gunicorn was only in ddtrace/contrib/gunicorn, so it wasn’t detected.

This PR ensures all integrations have a ddtrace/contrib/internal/{integration_name} directory, so this issue shouldn’t happen again.

cc: @wconti27